add more config doc descriptions (#3973)
owncloud/ocis: docs/services/thumbnails/_index.md

geekdocCollapseSection: true
---

## Abstract

> 
> This service provides an ocis extensions which generates thumbnails for image files.

Reviewer: ```suggestion

The thumbnail service generates thumbnails for image files.
```
